extension.json

```diff
@@ -1,5 +1,6 @@
 {
-	"name": "Persistent Page Identifiers",
```

Comment:
This makes the maintenance script think the extension is not loaded, because it looks for "PersistentPageIdentifiers". This PR makes the extension name human-readable by using i18n.